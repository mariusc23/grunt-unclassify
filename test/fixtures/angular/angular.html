<div ng-class="{ single : vm.variable }"></div>
<div ng-class="{ 'dashed-only' : vm.variable }"></div>
<div ng-class="{ 'dashed-first dashed-last' : vm.variable }"></div>
<div ng-class="{ 'dashed-first dashed-middle dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-single' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-only' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first exists-dashed-last' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first exists-dashed-middle exists-dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-dashed-first dashed-middle dashed-last' : vm.variable }"></div>
<div ng-class="{ 'dashed-first exists-dashed-middle dashed-last' : vm.variable }"></div>
<div ng-class="{ 'dashed-first dashed-middle exists-dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-dashed-first exists-dashed-middle dashed-last' : vm.variable }"></div>
<div ng-class="{ 'dashed-first exists-dashed-middle exists-dashed-last' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first dashed-middle exists-dashed-last' : vm.variable }"></div>

<div ng-class="only"></div>
<div ng-class="exists-only"></div>

<div ng-class="[first, middle, last]"></div>
<div ng-class="[first, exists-middle, last]"></div>
