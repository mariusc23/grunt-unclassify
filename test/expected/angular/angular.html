<div ng-class="{ single : vm.variable }"></div>
<div ng-class="{ 'dashed-only' : vm.variable }"></div>
<div ng-class="{ 'dashed-first dashed-last' : vm.variable }"></div>
<div ng-class="{ 'dashed-first dashed-middle dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-single' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-only' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first exists-dashed-last' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first exists-dashed-middle exists-dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-dashed-first' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-middle' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-last' : vm.variable }"></div>

<div ng-class="{ 'exists-dashed-first exists-dashed-middle' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-middle exists-dashed-last' : vm.variable }"></div>
<div ng-class="{ 'exists-dashed-first exists-dashed-last' : vm.variable }"></div>

<div></div>
<div ng-class="exists-only"></div>

<div ng-class="[first, middle, last]"></div>
<div ng-class="[first, exists-middle, last]"></div>
</div></div></div></div>
